aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -6,7 +6,7 @@ use crate::{
     utils::{bytes_to_string, is_string_layout, string_to_bytes},
 };
 use aptos_logger::error;
-// TODO: After aggregators_v2 branch land, consolidate these, instead of using alias here
+// TODO[agg_v2](cleanup): After aggregators_v2 branch land, consolidate these, instead of using alias here
```

Comment:
we are aliasing a type from aptos-types here, because due to some other changes - I needed to move these types there. (but didn't want to refactor everything in the large stack of PRs we have